charts/housnav/values-dev.yaml

```diff
@@ -25,4 +25,17 @@ readinessProbe:
 networkPolicy:
   enabled: true
 
-resources: {}
+resources:
+  requests:
+    memory: "50Mi"
+    cpu: "25m"
+  limits:
+    memory: "100Mi"
+    cpu: "50m"
+
+vault:
+  secretPath: d6af69-nonprod/housnav-secret-dev
+  authPath: auth/k8s-silver
+  namespace: platform-services
+  role: d6af69-nonprod
```

Comment:
Is this implementation dynamic? this is currently nonprod but what about for the prod environment? 

vault:
  secretPath: d6af69-nonprod/housnav-secret-dev
  authPath: auth/k8s-silver
  namespace: platform-services
  role: d6af69-nonprod
  serviceAccountName: d6af69-vault

Reply:
When production is implemented, we will simply add value-prod.yaml with the necessary parameters for the production environment